test/cli/test_cs2cs_various.yaml

```diff
@@ -0,0 +1,1511 @@
+  in: 2 49 0
+  out: |
+    49dN	2dE 0.000
+- comment: Test cs2cs geographic -> geocentric using BETA2007.gsb grid
+  args: EPSG:4746 EPSG:4978
+  in: 50.5 10 0
+  out: |
+    4003461.55	705832.08 4898267.79
+- comment: Test cs2cs geocentric -> geographic using BETA2007.gsb grid
+  args: EPSG:4978 EPSG:4746
+  in: 4003461.55 705832.08 4898267.79
+  out: |
+    50d30'N	10dE -0.001
+- comment: Test cs2cs geographic -> geocentric, normally using BETA2007.gsb grid but missing
+  tmpdir:
+    copy:
+    - $PROJ_DATA/proj.db
+  env:
+    PROJ_DATA: $tmpdir
+  args: EPSG:4746 EPSG:4978
+  in: 50.5 10 0
+  out: |
+    4003461.37	705832.04 4898267.94
+- comment: Test cs2cs geocentric -> geographic, normally using BETA2007.gsb grid but missing
+  tmpdir:
+    copy:
+    - $PROJ_DATA/proj.db
+  env:
+    PROJ_DATA: $tmpdir
+  args: EPSG:4978 EPSG:4746
+  in: 4003461.37 705832.04 4898267.94
+  out: |
+    50d30'N	10dE -0.002
+- comment: Test cs2cs geographic -> geocentric outside BETA2007.gsb grid
+  args: EPSG:4746 EPSG:4978
+  in: 49 2 0
+  out: |
+    4189881.02	146313.87 4790558.75
+- comment: Test cs2cs geocentric -> geographic outside BETA2007.gsb grid
+  args: EPSG:4978 EPSG:4746
+  in: 4189881.02 146313.87 4790558.75
+  out: |
+    49dN	2dE 0.002
+- comment: Test Similarity Transformation (example from EPSG Guidance Note 7.2)
+  args: -d 3 EPSG:23031 EPSG:25831 -E
+  in: 300000 4500000
+  out: |
+    300000 4500000	299905.060	4499796.515 0.000
+- comment: Test inverse of Similarity Transformation (example from EPSG Guidance Note 7.2)
+  args: -d 3 EPSG:25831 EPSG:23031 -E
+  in: 299905.060 4499796.515
+  out: |
+    299905.060 4499796.515	300000.000	4500000.000 0.000
+- comment: Test Similarity Transformation through CompoundCRS
+# Cf https://github.com/OSGeo/PROJ/issues/3854#issuecomment-1689964773
+  args: -d 3 EPSG:3912 EPSG:3794 -E
+  in: 477134.28 95134.21
+  out: |
+    477134.28 95134.21	476763.303	95620.222 0.000
+- args: -d 3 EPSG:3912+EPSG:5779 EPSG:3794+EPSG:8690 -E
+  in: 477134.28 95134.21 5
+  out: |
+    477134.28 95134.21 5	476763.303	95620.222 5.000
+- comment: Test --s_epoch
+  args: -d 8 --s_epoch 2022 "ITRF2014" "GDA2020" -E
+  in: -30 150
+  out: |
+    -30 150	-30.00000099	149.99999956 0.00031880
+- comment: Test --t_epoch
+  args: -d 8 --t_epoch 2022 "GDA2020" "ITRF2014" -E
+  in: -30 150
+  out: |
+    -30 150	-29.99999901	150.00000044 -0.00031879
+- args: >
+    +proj=topocentric +datum=WGS84 +X_0=-3982059.42 +Y_0=3331314.88 +Z_0=3692463.58 +no_defs +type=crs +to
+    EPSG:4978 -d 3 -E
+  in: 0 0 0
+  out: |
+    0 0 0	-3982059.420	3331314.880 3692463.580
+- comment: >
+    Test cs2cs EPSG:5488 (RGAF09) to EPSG:4559+5757 (RRAF 1991 / UTM zone 20N + Guadeloupe 1988 height)
+
+    Check that we use the horizontal transformation for Guadeloupe (RRAF 1991 to RGAF09 (2)), and not the one for Martinique
+  skipif: env.get("TIFF_ENABLED") == "NO"
```

Comment:
here's an example to skip a test if a TIFF_ENABLED is NO, otherwise it is not skipped.